### PR TITLE
Atom button

### PR DESCRIPTION
**Pull request by shaik-sohail**
Button with custom variants added. New variants are
#### 2.createTemplate
image
image

### CODE REVIEW

File: src/theme/index.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+let theme = createTheme({
+  palette: {
+    primary: {
+      main: '#2da6ca',
+    },
+  },
+  components: {
+    MuiButton: {
+      variants: [
```
Comment (inigorajv) [inline src/theme/index.tsx]: Shouldn't be creating variants(template, createTemplate) in Theme File.
Need to create using (sx or styles or styled Component) in pages.

Comment (shaik-sohail) [reply src/theme/index.tsx]: Removed custom variant

File: src/components/Atoms/Button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import {
+  Button as MuiButton,
+  ButtonProps as MuiButtonProps,
+  styled,
+} from '@mui/material'
+interface iProps extends MuiButtonProps {}
+const CustomizedButton = styled(MuiButton)(() => ({}))
+const CutomButton: React.FC<iProps> = (props) => {
```
Comment (inigorajv) [inline src/components/Atoms/Button/index.tsx]: check Spell

File: src/components/Atoms/Button/index.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  const { children } = props
+  return (
+    <>
```
Comment (inigorajv) [inline src/components/Atoms/Button/index.tsx]: fragment wrapping not required for single component.

File: src/components/Atoms/Button/index.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import {
+  Button as MuiButton,
+  ButtonProps as MuiButtonProps,
+  styled,
+} from '@mui/material'
+interface iProps extends MuiButtonProps {}
+const CustomizedButton = styled(MuiButton)(() => ({}))
```
Comment (inigorajv) [inline src/components/Atoms/Button/index.tsx:8]: Reason for Creating styled button?

File: src/components/Atoms/Button/index.tsx (hunk 3 above)
Comment (shaik-sohail) [reply src/components/Atoms/Button/index.tsx]: removed

File: src/components/Atoms/Button/index.tsx (hunk 2 above)
Comment (shaik-sohail) [reply src/components/Atoms/Button/index.tsx]: Modified

**Review by inigorajv — APPROVED** [review]

**Pull request merged**